.github/workflows/push_main.yaml

```diff
@@ -6,6 +6,13 @@ on:
       - main
 
 jobs:
+
+  lint-report:
+    uses: ./.github/workflows/lint-report.yaml
+
+  static-analysis:
+    uses: ./.github/workflows/static-analysis.yaml
+
```

Comment:
Just nitpicking, but you could merge both `push` workflows into one file and just run publish conditionally. That would reduce dupliacation.

Reply:
I agree, done :)